README.md

```diff
@@ -10,24 +10,50 @@ Inmanta Documentation on [Read the Docs](https://inmanta.com/resources/docs/)
 
 ## Features
 
-For now, just basic Syntax Coloring
 
-![Screenshot](images/screenshot.png)
+### Syntax coloring
+The extension makes your life easier by coloring syntax keywords or the Inmanta language.
 
-## Alpha Features
+![Syntax coloring screenshot](images/screenshot-syntax-coloring.png)
 
 ### Code navigation
+The extension allows you to navigate to the Definition of types used by using the shortcut `Ctrl + Click` when you hover the element.  And will show an overview of the Entity if you don't click.
 
-To enable ctrl-click for code navigation, take the following steps:
+![Navigation screenshot](images/screenshot-ctrl-click.png)
+
+### Compilation error reporting
+The extension will run a pre-compilation every time you save your file.  Making it easier for you to spot syntax and basic model errors.  When hovering the problematic bit, an explaination message is shown, and more details can be found in the **Output** panel (when selecting *Inmanta Server Language* in the expandable list on the right).
+
+![Error reporting screenshot](images/screenshot-error-reporting.png)
+
+### Model export
+The extension adds a button in the upper right corner of the editing panel of any inmanta model file (ending in `.cf`): **Export to server**.  This buttons allows you to easily export the current open model to an Inmanta server running.  Note that for this to work, you need to have previously configured the server, and added the following informations in the `.inmanta` configuration file at the root of the project:
+
+```
+[config]
+fact-expire=1800
+environment=f2f6ac46-10a2-4e75-897d-b91c51c78df2  # Replace this by the right environment uuid
+
+[compiler_rest_transport]
+host=10.0.0.102                                   # Replace this by the address of the host
+port=8888                                         # Replace this by the port on which the server is listening
+
+[cmdline_rest_transport]
+host=10.0.0.102                                   # Replace this by the address of the host
+port=8888                                         # Replace this by the port on which the server is listening
+```
+
+## Installation
+
+To enable ctrl-click for code navigation, a few extra steps are required:
```

Comment:
is this still required?

Reply:
It seems like it is not, indeed.  If not input is given for the `Python Path`, a default one is created and everything will work.

Reply:
I updated the documentation accordingly here: https://github.com/inmanta/vscode-inmanta/pull/43/commits/8d43418d91ec5cf21b0f6846210a337521fd6668

 - Removed the installation part
 - Added a note below the features requiring the LS, explaining how to enable it.